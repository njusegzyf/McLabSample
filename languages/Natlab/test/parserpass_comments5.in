%comment1
a = 5;
%comment2
b = 5;