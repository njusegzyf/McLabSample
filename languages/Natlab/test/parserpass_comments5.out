1 10
4 6

%comment1
a = 5;

%comment2
b = 5;
